.npmrc

```diff
@@ -0,0 +1 @@
+registry = https://registry.npmjs.org/
```

Comment:
_I think_ you need to `yarn install` again... I still see `yarnpkg.com` in the lockfile and this should affect that

Reply:
In https://github.com/goodeggs/resource-client/pull/18/commits/e36b8c8e5a7d47351325315bc6a9a326d18f3262 I deleted `yarn.lock` and re-ran `yarn install` - we're back to `https://npm.goodeggs.com` in the lockfile

Reply:
To go with this change, do we need to add additional information about how to authorize Travis to install packages?

The [goodeggs-json-schema-validator](https://github.com/goodeggs/goodeggs-json-schema-validator) and [fake-eggs](https://github.com/goodeggs/fake-eggs) packages have yarn.lock files that point to `yarnpkg.com` and have simpler `.travis.yml` files than [goodeggs-ui](https://github.com/goodeggs/goodeggs-ui/blob/master/.travis.yml) which has a secure token. Should I add a secure token? I think I saw that in the docs.